src/python/pants/option/global_options.py

```diff
@@ -1381,7 +1381,11 @@ class BootstrapOptions:
         deprecation_start_version="2.15.0.dev2",
         removal_version="2.16.0.dev1",
         removal_hint=softwrap(
-            "Remote auth plugins should now provide the function by implementing an entry point called remote_auth. If you are developing a plugin, switch to using an entry point. If you are only consuming a plugin from someone else, you can delete the remote_auth_plugin option and now only need the plugin to be included in [GLOBAL].plugins"
+            """
+            Remote auth plugins should now provide the function by implementing an entry point called remote_auth. 
+            If you are developing a plugin, switch to using an entry point. 
```

Comment:
Do you intend for these to be distinct paragraphs? If so, they need a blank line. `softwrap` concatenates everything into the same paragraph otherwise. Which means the last line below should be written over 2 lines so we avoid horizontal scrolling